packages/testkit/src/mock-ckb-jsonrpc.ts

```diff
@@ -43,11 +43,11 @@ export function createCKBMockRPC(options: Options): Express {
   });
 
   server.addMethod("get_tip_block_number", () => {
-    if(blocks.length < 1) {
-      return 'null'
+    if (blocks.length < 1) {
+      return "null";
```

Comment:
why is it a null string instead of a null?

Reply:
changed to use null